src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelDataTypeResolver.java

```diff
@@ -75,6 +75,12 @@ protected void resolveProperty(CodegenProperty property, ApiResourceBuilder apiR
         } else {
             property.vendorExtensions.put("json-name", property.name);
         }
+        if(CodegenUtils.isPropertySchemaEnum(property)
+            && !property.dataType.contains(ApplicationConstants.ENUM)
+            && !property.dataType.contains("Instance.")) {
+                property.baseType = ApplicationConstants.ENUM + property.baseType;
+                property.dataType = ApplicationConstants.ENUM + property.dataType;
+        }
```

Comment:
You need to make sure for such enums, the created enum name should be same as of defined in twilio-oai